src/main/java/ch/njol/skript/expressions/ExprChatRecipients.java

```diff
@@ -113,8 +110,28 @@ public void change(Event event, @Nullable Object[] delta, ChangeMode mode) {
 			case REMOVE_ALL:
 			case RESET:
 			case DELETE:
-				((AsyncPlayerChatEvent) event).getRecipients().clear();
+				if (event instanceof AsyncPlayerChatEvent) {
+					((AsyncPlayerChatEvent) event).getRecipients().clear();
+				} else {
+					((BroadcastMessageEvent) event).getRecipients().clear();
+				}
 				break;
 		}
 	}
+
+	@Override
+	public boolean isSingle() {
+		return false;
+	}
+
+	@Override
+	public Class<CommandSender> getReturnType() {
+		return CommandSender.class;
+	}
+
+	@Override
+	public String toString(@Nullable Event event, boolean debug) {
+		return "chat recipients";
```

Comment:
This suggestion or you can create a field to indicate `isBroadcast` for proper toString
```suggestion
		return "chat/broadcast recipients";
```

Reply:
The toString must match the syntax. broadcast recipients is not a valid syntax.

Reply:
Good point, then I would recommend updating the syntax to add broadcast `[broadcast|chat][( |-)]recipients` and `chat/broadcast recipients` in toString is just okay here since you can't always get `isBroadcast`